requirements/base.txt

```diff
@@ -8,3 +8,4 @@ django-modern-rpc==0.11.1
 django-simple-history==2.7.3
 jira==2.0.0
 Markdown==3.1.1
+python-redmine>=2.2.1
```

Comment:
Use `==` here. @pyup-bot will update it automatically when new version is available.